app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/WizNavCheckTokenActivity.java

```diff
@@ -47,6 +46,12 @@ public void onClickNext(View next) {
         startLoading();
     }
 
+    public void onClickTUMOnline(View next) {
+        Intent intent = new Intent(Intent.ACTION_VIEW);
+        intent.setData(Uri.parse("http://campus.tum.de"));
```

Comment:
Please move that string constant to Const.kt

Reply:
Fixed.